x-pack/plugins/uptime/server/graphql/unsigned_int_scalar/resolvers.ts

```diff
@@ -19,6 +19,26 @@ const unsignedIntegerScalar = new GraphQLScalarType({
     if (parsed < 0) {
       return null;
     }
+    return parsed;
+  },
```

Comment:
So, I'm thinking we don't need to add all this. We can simply return a string for the one place where this type would be needed. Check out this commit: https://github.com/justinkambic/kibana/commit/1222b43ca937cdcf1eced58b1dd2900397fcf8bb.

In there I modify the schema and front end as needed and update the API tests.